src/core/blueprint/operators/http.rs

```diff
@@ -94,7 +94,20 @@ pub fn compile_http(
                     is_list,
                     dedupe,
                 })
+            };
+
+            if let Some(select) = &http.select {
+                let dynamic_value = match DynamicValue::try_from(select) {
+                    Ok(dynamic_value) => dynamic_value.prepend("args"),
+                    Err(_) => {
```

Comment:
We should keep the original error also. Try using `Valid::fail_with`.

NOTE: Tests are missing for this branch.